src/game/rendering_graph_node.c

```diff
@@ -305,25 +305,27 @@ extern const Gfx init_rsp[];
 
 #ifdef OBJECTS_REJ
 void switch_ucode(s32 ucode) {
+    Gfx *tempGfxHead = gDisplayListHead;
+
     // Set the ucode and RCP settings
     switch (ucode) {
         default: // GRAPH_NODE_UCODE_DEFAULT
         case GRAPH_NODE_UCODE_DEFAULT:
-            gSPLoadUcodeL(gDisplayListHead++, gspF3DZEX2_NoN_PosLight_fifo); // F3DZEX2_PosLight
+            gSPLoadUcodeL(tempGfxHead++, gspF3DZEX2_NoN_PosLight_fifo); // F3DZEX2_PosLight
             // Reload the necessary RSP settings
-            gSPDisplayList(gDisplayListHead++, init_rsp);
+            gSPDisplayList(tempGfxHead++, init_rsp);
             break;
         case GRAPH_NODE_UCODE_REJ:
             // Use .rej Microcode, skip sub-pixel processing on console
             if (gEmulator & EMU_CONSOLE) {
-                gSPLoadUcodeL(gDisplayListHead++, gspF3DLX2_Rej_fifo); // F3DLX2_Rej
+                gSPLoadUcodeL(tempGfxHead++, gspF3DLX2_Rej_fifo); // F3DLX2_Rej
             } else {
-                gSPLoadUcodeL(gDisplayListHead++, gspF3DEX2_Rej_fifo); // F3DEX2_Rej
+                gSPLoadUcodeL(tempGfxHead++, gspF3DEX2_Rej_fifo); // F3DEX2_Rej
             }
             // Reload the necessary RSP settings
-            gSPDisplayList(gDisplayListHead++, init_rsp);
+            gSPDisplayList(tempGfxHead++, init_rsp);
             // Set the clip ratio (see init_rsp)
-            gSPClipRatio(gDisplayListHead++, FRUSTRATIO_2);
+            gSPClipRatio(tempGfxHead++, FRUSTRATIO_2);
             break;
```

Comment:
here too. bro did not check if the rom works

Reply:
All of a sudden, that deconstructor idea seems like a good one...

Reply:
the rom literally does work even with these mistakes :sob:

Reply:
Automated build and test system when

Reply:
if we ever separate ourselves from needing the rom to build (unlikely) i can spin up a github actions :smirk_cat:

Reply:
y'all are REALLY good at scope creeping out of nowhere arent you....

Reply:
how is verifying a build under many common define combinations scope creep (especially when its definitely going into another PR) .....

Reply:
"oh i forgot to change this type to s32 instead of s16. this reminds me of how we need to add gcc c++ extension lua pc port auto jenkins to hacker" :trollface:

Reply:
(joke)